src/mpags-cipher.cpp

```diff
@@ -84,73 +56,64 @@ int main(int argc, char* argv[])
         return 0;
     }
 
-    // Initialise variables
-    char inputChar{'x'};
+    char in_char{'x'};
     std::string inputText;
-
-    // Read in user input from stdin/file
+    std::string outputText;
+    
+    // Read in user input from stdinfile
     // Warn that input file option not yet implemented
     if (!inputFile.empty()) {
-        std::cerr << "[warning] input from file ('" << inputFile
-                  << "') not implemented yet, using stdin\n";
-    }
+        std::ifstream inputStream{inputFile};
 
-    // loop over each character from user input
-    while (std::cin >> inputChar) {
-        // Uppercase alphabetic characters
-        if (std::isalpha(inputChar)) {
-            inputText += std::toupper(inputChar);
-            continue;
+        if (!inputStream.good()){
+            std::cerr << "unable to open file" << std::endl;
+            return 1;
         }
 
-        // Transliterate digits to English words
-        switch (inputChar) {
-            case '0':
-                inputText += "ZERO";
-                break;
-            case '1':
-                inputText += "ONE";
-                break;
-            case '2':
-                inputText += "TWO";
-                break;
-            case '3':
-                inputText += "THREE";
-                break;
-            case '4':
-                inputText += "FOUR";
-                break;
-            case '5':
-                inputText += "FIVE";
-                break;
-            case '6':
-                inputText += "SIX";
-                break;
-            case '7':
-                inputText += "SEVEN";
-                break;
-            case '8':
-                inputText += "EIGHT";
-                break;
-            case '9':
-                inputText += "NINE";
-                break;
+        while(inputStream >> in_char){
+            inputText += transformChar(in_char);
         }
-
-        // If the character isn't alphabetic or numeric, DONT add it
     }
 
-    // Print out the transliterated text
+    else{
+        while(std::cin >> in_char){
+
+            inputText += transformChar(in_char);
+        }
+
+    }
+    outputText = runCaesarCipher(inputText, key, encrypt, decrypt);
```

Comment:
Better to declare and initialise this variable here, rather than declaring up on line 61 and then only doing something with it finally here:
```suggestion
    std::string outputText { runCaesarCipher(inputText, key, encrypt, decrypt) };
```